### PR TITLE
FOCUS dataset consistency review

### PR DESCRIPTION
**Pull request by aqu-erp**
Resolving #472 

### Acceptance
- [ ] select a term to used to describe a FOCUS compliant dataset
- [ ] defining term in glossary
- [ ] maintain consistent usage of term across specification and changelog, etc.
- [ ] add glossary anchors where needed
- [ ] confirm convention for referring to a generic potentially noncompliant dataset from a provider

### Proposed term and definition
`FOCUS Dataset`
 `A structured collection of cost and usage data that meets or exceeds the Basic compliance criteria of FOCUS.`

---

### Open Questions

When to retain use of `billing data`
 - When used in general context of FinOps data at large

#### Examples
Overview.md
>FOCUS aims to establish a community-driven specification for consumption-based _billing data_. 

>_Billing data_ generators: Infrastructure and services providers that bill based on consumption

Readme.md
>The FOCUS spec will make it easier to merge multiple _billing data_ sources together

---

### Notes

- Term replaces instances of `-`, `billing data`, and `cost data` in normative statements
- Term does not refer to dataset supplied/created/provided by the spec
- Term refers to dataset that is being described by normative statements
   - a dataset that meets certain compliance criteria

### CODE REVIEW

**Review by flanakin — APPROVED** [review]

File: specification/columns/availabilityzone.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 An [*availability zone*](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
-The AvailabilityZone column is RECOMMENDED to be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
+The AvailabilityZone column is RECOMMENDED to be present in a FOCUS dataset when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```
Comment (ijurica) [inline specification/columns/availabilityzone.md:5]: I assume we need to add links to the term in the glossary.

Not sure if we should use `<a name="glossary:FOCUS-dataset"><b>FOCUS Dataset</b></a>` or `<a name="glossary:focus-dataset"><b>FOCUS Dataset</b></a>`. For comparison, we've used `<a name="glossary:sku"><b>SKU</b></a>`, If the link/syntax works, I guess it doesn't matter.

```suggestion
The AvailabilityZone column is RECOMMENDED to be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Comment (ijurica) [reply specification/columns/availabilityzone.md:5]: btw, @aqu-erp great job!

**Comment by ijurica** [comment]
We've mentioned that if native/custom columns are present in the FOCUS dataset, they must be prefixed with 'x_'. Do we need to emphasize that, **if present, custom columns must also be listed in the Metadata - Schema - Column Definition**. 

By the way, we've specified that each column is represented with a Column Name. Would Column IDs be more useful in this context?

**Comment by jpradocueva** [comment]
### Action Item
Maintainers-[#474](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/474): Andrew to lead the review and discussions in Task Force 3.

File: specification/columns/availabilityzone.md (hunk 1 above)
Comment (flanakin) [reply specification/columns/availabilityzone.md:5]: Great catch!

**Comment by jpradocueva** [comment]
Action Items agreed on TF-3, July 12 call: 
- [ ] Maintainers-[#474](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/474) : Andrew will lead the review and discussions in Task Force 3.
- [ ] TF3- [#474](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/474) All members to review and provide comments on the PR 
- [ ] TF3- [#474](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/474) Members: Discuss and finalize the inclusion of native provider columns in the FOCUS dataset definition in the glossary.

**Review by udam-f2 — APPROVED** [review]

**Comment by jpradocueva** [comment]
This PR was approved by the group during the Maintainers meeting on August 1.

**Pull request merged**